caste.go

```diff
@@ -1496,3 +1496,29 @@ func trimZeroDecimal(s string) string {
 	}
 	return s
 }
+
+func truncateDecimals(s string) string {
```

Comment:
Looks like this function could use a test?

Reply:
Alright, I have added it.